ocs_ci/ocs/ui/add_replace_device_ui.py

```diff
@@ -31,11 +31,9 @@ def add_capacity_ui(self):
             self.do_click(self.add_capacity_ui["ocs_operator"])
             self.do_click(self.add_capacity_ui["storage_cluster_tab"])
         self.do_click(self.add_capacity_ui["kebab_storage_cluster"])
+        self.take_screenshot()
```

Comment:
It will be a good idea to simply print the name of the storage class visible on the UI, gp2 for AWS, thin for VMware, etc. Simply fetch the text.

Reply:
It is problematic because the UI team changes the storage class name often.